php/Terminus/Helpers/Input.php

```diff
@@ -302,31 +303,32 @@ public static function orgId(array $arg_options = array()) {
 
     $arguments = $options['args'];
     $key       = $options['key'];
-    if (!isset($arguments[$key]) && isset($_SERVER['TERMINUS_ORG'])) {
-      $arguments[$key] = $_SERVER['TERMINUS_ORG'];
-    }
-
-    $org_list = self::orgList($options);
-    $flip    = array_flip($org_list);
+    $org_list  = self::orgList($options);
     if (isset($arguments[$key])) {
-      if (isset($flip[$arguments[$key]])) {
-        return $flip[$arguments[$key]];
-      } elseif (isset($org_list[$arguments[$key]])) {
-        return $arguments[$key];
-      } elseif (in_array($arguments[$key], self::$NULL_INPUTS)
-        || !empty($arguments)
-      ) {
+      if ($id = array_search($arguments[$key], $org_list)) {
+        return $id;
+      }
+      return $arguments[$key];
+    } else if (isset($_SERVER['TERMINUS_ORG'])) {
+      return $_SERVER['TERMINUS_ORG'];
```

Comment:
Question: should we be grabbing these constants out of $_SERVER or are they getting defined in the [constants](https://github.com/pantheon-systems/cli/blob/15537cde64ccc6039ae9ea1d06b57018df27225d/php/utils.php#L107-L124)... though I guess these aren't _required_ for running terminus. 

I think grabbing out of $_SERVER is fine, but I want to at least plant an seed for thinking about how we could further consolidate our config vars.